### DISCUSSION TITLE
php artisan shopper:admin

### DISCUSSION THREAD

**Discussion by IgorDePaula** [shopperlabs/framework#104] [General]
![Captura de tela de 2022-04-17 20-16-17](https://user-images.githubusercontent.com/2833993/163735747-f1e14275-c22b-4806-8140-d3d23bc24129.png)

Laravel 9
sqlite database
php 8.1
ubuntu 21.10

The install command created many table, but not change the default user table.

**Comment by IgorDePaula** [comment]
![Captura de tela de 2022-04-17 20-21-09](https://user-images.githubusercontent.com/2833993/163735866-17d6c624-8590-4b27-b9ec-b18a21ef2b2f.png)

screen after login. I update the table put an admin user. But not working.

**Reply by gabolera** [reply]
I solved one error in installation steps (similar your first printscreen but with column "gender") but now i stopped in this error and don't know how to solve, did you manage to solve it?

**Comment by IgorDePaula** [comment]
![Captura de tela de 2022-04-17 20-36-48](https://user-images.githubusercontent.com/2833993/163736491-573ddc5d-5c63-414e-81fb-a0eb0cef86fd.png)
Your model havent last_name first_name on your own model.

**Comment by detygon** [comment]
Hey, have you run `php artisan shopper:install`?. It's sometimes easy to miss these things. Follow carefully the instructions [here](https://laravelshopper.dev/installing) and let us know how it goes

**Comment by IgorDePaula** [comment]
Hey, Yes, I does all instructions. But how i can see, the code is not complete.

**Comment by gabolera** [comment]
i'm see now that exists pending a pull request more than 20 days #129, and this PR solve 2 bugs on execute `php artisan shopper:admin` in columns "first_name" and "gender" that not defined with nullable but is waiting approve merge
